### ISSUE TITLE
Invalid isbns are returned as valid (is_isbn13), but mask works.

### ISSUE THREAD

**Issue by EmilStenstrom** [xlcnd/isbnlib#15]
```
>>> import isbnlib
>>> isbn = "9786610326266"
>>> isbnlib.is_isbn13(isbn)
True
>>> isbnlib.mask(isbn)
None
```

This is an invalid ISBN according to http://isbn.org/ISBN_converter

**Comment by xlcnd** [comment]
9786610326266 is a valid ISBN!

```
isbn> meta 9786610326266
Type:      BOOK
Title:     Teaching And Learning In Further Education: Diversity And Change
Author:    Prue Huddlestone and Lorna Unwin
ISBN:      9786610326266
Year:      1997
Publisher: Routledge
```

However is not in the ISBN registry! Sometimes this happens but only with ebooks...

I will research further.

By the way, what version of ``isbnlib`` are you using?

**Comment by xlcnd** [comment]
In the *isbn.org* database, blocks starting with 978-66..., are as not been issued yet!

However, they here already used. There are lots and lots of errors like this. Blame the publishers.

There are some confusion with the concepts too. **Valid ISBN** should be an ISBN that was built according with the rules, this is distinct from **issued ISBN** an ISBN that was already issued to a publisher (this is the usage of the libraries too).

**Comment by xlcnd** [comment]
It is very easy to build an ``issued`` flag:

```python
issued = True if mask(isbn) else False
```

**Comment by EmilStenstrom** [comment]
Hi,

I'm using the latest released version of isbnlib: 3.4.7. 

Thanks for clarifying the difference between valid and issued (maybe also add this to a quickstart section?). I simply assumed that those two concepts was the same thing. I will use mask to make sure my ISBNs are also issued. Thanks!
